### ISSUE TITLE
Elasticsearch support

### ISSUE THREAD

**Issue by alemhnan** [adrai/node-cqrs-eventdenormalizer#70]
Apologies for the maybe silly question, is Elasticsearch supported in the denormalization? 

I see that I can specify settings for a collection, but I don't see, at least in the docs, where to set up the connection to Elasticsearch.

**Comment by adrai** [comment]
Never used it, but you can specify the connection settings in the repository options.

The repository options are these: https://github.com/adrai/node-viewmodel/blob/master/README.md (there are 2 different elasticsearch versions)

**Comment by nanov** [comment]
We use it in production with Elasticsearch 6.1, you should specify your index mappings, settings and ca. in the collection definition and then everything should work pretty much the same.

**Comment by tommiii** [comment]
I've some doubts about defineCollection and defineViewBuilder for elasticsearch, can you post some example?

**Comment by nanov** [comment]
Something like [this](https://gist.github.com/nanov/15ce1a154c30361ac5ed641014b5186f)?

**Comment by tommiii** [comment]
Yes, thanks. Now everything is working. Thank you.

**Comment by tommiii** [comment]
Yes! thanks, now is working. I didn't get how to set the _index and the _type. Looks like that the lib create a string composed by index.collectionName for _index and _type. Is there a method for set them?

**Comment by nanov** [comment]
Well, the whole thing was written with Elasticsearch 6.X in mind, this means that each [index can have only one type and the types will be removed as a whole from Elasticsearch in the future](https://www.elastic.co/guide/en/elasticsearch/reference/6.x/removal-of-types.html), that is the reason why we left only the index is configurable.

**Comment by tommiii** [comment]
I wasn't aware of this, makes sense. Thank you.
